nitric/proto/resources/v1/resources.proto

```diff
@@ -75,6 +79,19 @@ message KeyValueStoreResource {
 message SecretResource {
 }
 
+message JobResourceRequirements {
+  // The number of CPUs to allocate for the job
+  int64 cpus = 1;
+  // The amount of memory to allocate for the job
+  int64 memory = 2;
+  // The number of GPUs to allocate for the job
+  int64 gpus = 3;
```

Comment:
Should these be `double`?

Reply:
That's a good question actually. For values that might be fractional e.g. CPUs possibly. For memory and GPU probably not so much (I haven't seen timesliced provisioned config of either).

Might be worth going over each of the configs to figure that out.